modules/Microsoft.Network/connections/readme.md

```diff
@@ -14,7 +14,7 @@ This template deploys a virtual network gateway connection.
 | Resource Type | API Version |
 | :-- | :-- |
 | `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
-| `Microsoft.Network/connections` | [2021-05-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-05-01/connections) |
+| `Microsoft.Network/connections` | [2022-01-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/connections) |
```

Comment:
Please update to latest version documented in ARR so that the link is mapped to the specific version used and not to generic latest (which can change over time)

Reply:
Updated to older API version